tests/unit_tests/test_wave_06.py

```diff
@@ -261,11 +253,8 @@ def test_swap_best_by_category_no_other_match_is_false():
         their_priority="Decor"
     )
 
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
-    # Assertions should check:
-    # - That result is falsy
-    # - That tai and jesse's inventories are the correct length
-    # - That all the correct items are in tai and jesse's inventories
+    assert result is False
+    assert len(tai.inventory) == 3
+    assert len(jesse.inventory) == 3
+    assert item_b in tai.inventory
+    assert item_e in jesse.inventory
```

Comment:
What other assertions could you write to confirm that the other inventory items weren't affected by invoking `swap_by_newest`?